### DISCUSSION TITLE
Allowing Syncing Trophies From Other Platforms

### DISCUSSION THREAD

**Discussion by FOSSilizedDaemon** [RetroAchievements/RAWeb#1597] [Ideas]
One of the constant things I am noticing many people say they wish RetroAchievements supported is syncing trophies from other platforms such as Steam, GOG, PSN, etc. I personally have wanted to have one platform where I can view and organize all of my gaming progress from across the various platforms. Seeing as it will be a long time before we get support for more recent platforms would it all be possible to allow people to sync achievements from other verified platforms? I feel that, seeing as RetroAchievements is the only open source gaming platform really focusing on stuff like this, it would be very valuable to add this support. (Apologies if this has already been asked for in the past. I tried searching both issues and pull requests and found nothing).

**Comment by meleu** [comment]
This doesn't seem aligned with the [RetroAchievements mission](https://docs.retroachievements.org/Users-Code-of-Conduct/#mission-statement).

**Comment by RALordAndrew** [comment]
There are existing sites that offer this service already. Exophase can track achievements across multiple platforms including Steam, GOG, PlayStation, Xbox, RetroAchievements and more.
